### PR TITLE
i18n(es): Update `talks.mdx`, `netlify.mdx` & `github.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### Description (required)

`talks.mdx` https://github.com/withastro/docs/commit/3193d5c8d756b77cfe9d676badbfff92221eda74

`netlify.mdx` https://github.com/withastro/docs/commit/88585762e74a691ecdc2186267ff8e05b583eaed

`github.mdx` https://github.com/withastro/docs/commit/76c97f7fa29839ac31a77aae4180f691e5fe4077

### CODE REVIEW

File: src/content/docs/es/guides/integrations-guide/netlify.mdx [hunk 1]
```diff
@@ -166,13 +166,33 @@ export default defineConfig({
 });
 ```
 
-:::caution
-Este adaptador no soporta las propiedades de configuración `image.domains` e `image.remotePatterns` en tu configuración de Astro. Para [especificar rutas remotas para Netlify Image CDN](https://docs.netlify.com/image-cdn/overview/#remote-path), usa el campo `remote_images` en `netlify.toml`.
-:::
+If you are using images hosted on another domain, you must authorize the domain or URL patterns using the [`image.domains`](/en/reference/configuration-reference/#imagedomains) or [`image.remotePatterns`](/en/reference/configuration-reference/#imageremotepatterns) configuration options:
```
Comment (mingjunlu) [inline src/content/docs/es/guides/integrations-guide/netlify.mdx:169]: Found some typos here

```suggestion
If you are using images hosted on another domain, you must authorize the domain or URL patterns using the [`image.domains`](/es/reference/configuration-reference/#imagedomains) or [`image.remotePatterns`](/es/reference/configuration-reference/#imageremotepatterns) configuration options:
```

**Comment by Waxer59** [comment]
Done, thanks for the review @mingjunlu 🤠!

**Comment by Waxer59** [comment]
Thanks again 🤠!

**Review by AitorMT — APPROVED** [review]
LGTM!

**Review by dreyfus92 — APPROVED** [review]
Thanks everyone, LGTM ✅

**Pull request merged**